code/game/objects/items/cards_ids.dm

```diff
@@ -73,10 +73,16 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	var/id_type_name = "identification card"
 	var/mining_points = 0 //For redeeming at mining equipment vendors
+	///The stuff that makes you open doors and shit
 	var/list/access = list()
-	var/registered_name = null // The name registered_name on the card
+	///Access that cannot be removed by the ID console. Do not add access levels that are actually visible in the console here if a HoP knowing what kind of ID he's modifying is a concern.
+	var/list/StickyAccess = list()
```

Comment:
Could yo use `snake_case` same as all the other variables?
And could you turn this into a lazy list?
Feel free to poke me on discord if you don't know what that is (there should be a pinned message in the coder channel explaining it).

Reply:
Done.